### DISCUSSION TITLE
Why using proxy tries to hit http://burpsuite

### DISCUSSION THREAD

**Discussion by ferasdour** [projectdiscovery/katana#1053] [Q&A]
When trying to access a site, like this:
`echo "example.com"|/root/.pdtm/go/bin/katana -nc -silent -c 30 -p 10 -jc -kf  -no-sandbox -proxy http://172.25.128.1:8080`
The request received by the proxy is just this:
`GET / HTTP/1.1
Host: burpsuite
User-Agent: Go-http-client/1.1
Accept-Encoding: gzip
`
I'm getting in my proxy (caido and zap) a request to "burpsuite:80" and it doesn't really make sense why. Can someone help or tell me i'm doing it wrong?

**Comment by bill-e-ghote** [comment]
Same here. Why is this happening?

**Comment by ferasdour** [comment]
I think i found it, it's because it pulls in the proxy util from the project discovery utils and does an isburp check:

https://github.com/projectdiscovery/katana/blob/f8486d4535469a823ce661e28fe8a6db0cb0c1c0/pkg/engine/common/http.go#L47
https://github.com/projectdiscovery/utils/blob/main/proxy/burp.go#L14

I don't understand why thats a thing exactly, and the commit doesn't clearly explain the need for it:
https://github.com/projectdiscovery/utils/commit/47010959d14fac0566d11db12bd038ced32c6d2b

but I an see where in the code it happens at least, so that's.... something.
